### PR TITLE
Setting up directories and adding required player files.

### PR DESCRIPTION
**Pull request by juliareichel**
-Set up the directory tree and copied over HTML and javascript player files

### CODE REVIEW

File: SIMID/examples/maps/player/simid_player.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
+ <html>
```
Comment (damargulis) [inline SIMID/examples/maps/player/simid_player.html:1]: You shouldn't need any of the player/ files.

You can continue to use the player hosted from the simid repo, and only host your creatives here

Comment (damargulis) [inline SIMID/examples/maps/player/simid_player.html:1]: Add some text to the maps/README file. Can be short, just explain what the creative will be and what files will go in that dir

Comment (juliareichel) [reply SIMID/examples/maps/player/simid_player.html]: done!

Comment (juliareichel) [reply SIMID/examples/maps/player/simid_player.html]: deleted them

**Comment by juliareichel** [comment]
> This might be a good start, but for the appengine to work correctly, the folders, tests and html/js documents should be organized very differently than

File: maps/simid_protocol.js [hunk 2]
```diff
@@ -0,0 +1,388 @@
+/**
+ * Contains logic for sending mesages between the SIMID creative and the player.
+ * Note: Some browsers do not support promises and a more complete implementation
+ *       should consider using a polyfill.
```
Comment (damargulis) [inline maps/simid_protocol.js:4]: Add a note giving the url where this file originally came from.

File: maps/README.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+The goal of this project is to create a sample interactive ad creative for SIMID. 
+Our aim is to design a creative using the Google Maps API that provides users a 
+map of nearby locations where they could find the advertised product.
+
+The files in this directory include:
```
Comment (damargulis) [inline maps/README.txt:5]: You don't need to actually list the files, just a description of in general what will go here.
I.e. something like `Contains files for SIMID creative that will populate the iframe`

File: maps/README.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+The goal of this project is to create a sample interactive ad creative for SIMID.
```
Comment (damargulis) [inline maps/README.txt:1]: Looks like you accidentally added .README.txt.swp, be sure to remove

You can add the line `*.swp` to your .gitignore file and that should prevent accidentally adding swp files

Comment (juliareichel) [reply maps/README.txt:1]: Deleted.

File: maps/README.txt (hunk 3 above)
Comment (juliareichel) [reply maps/README.txt]: Got it! Fixed

File: maps/simid_protocol.js (hunk 2 above)
Comment (juliareichel) [reply maps/simid_protocol.js:4]: Added

**Pull request merged**